quarto/_extensions/countdown/assets/countdown.css

```
@@ -0,0 +1,169 @@
+.countdown {
+  --_running-color: var(--countdown-color-running-text, rgba(0, 0, 0, 0.8));
```

Revealjs has different slide scaling mechanism, so I want to change the default margin in revealjs land down to zero. The next three suggestions should handle that.

```suggestion
.countdown {
  --_margin: 0.6em;
  --_running-color: var(--countdown-color-running-text, rgba(0, 0, 0, 0.8));
```

---

Should we try to split this out into a subclass? `.reveal .countdown` ?